source/NVDAObjects/IAccessible/__init__.py

```diff
@@ -1289,17 +1289,20 @@ def _get_selectionContainer(self):
 			return self.table
 		return super(IAccessible,self).selectionContainer
 
-	def _getSelectedItemsCount_accSelection(self,maxCount):
+	def _getSelectedItemsCount_accSelection(self, maxCount):

```

Comment:
If this is being changed, could this be updated

```suggestion
	def _getSelectedItemsCount_accSelection(self, maxCount: int) -> int:
```